packages/rpc-transport/src/index.ts

```diff
@@ -0,0 +1,5 @@
+export interface IJsonRpcTransport {
+    send<TParams, TResponse>(method: string, params: TParams): Promise<TResponse>;
+}
+
+export {};
```

Comment:
ES modules _have_ to export _something_ that's not just a type. In the next few PRs I'll be adding concrete implementations of transports, so this will get fleshed out.

Reply:
ah gotcha, thanks for the explanation!